kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -23,13 +25,32 @@ describe('Overview', () => {
     },
   ];
 
+  const renderComponent = ({
+    underReplicatedPartitions = 1,
+    inSyncReplicas = 1,
+    replicas = 1,
+  } = {}) =>
+    render(
+      <Overview
+        name={mockTopicName}
+        partitions={mockPartitions}
+        internal={undefined}
+        clusterName={mockClusterName}
+        topicName={mockTopicName}
+        clearTopicMessages={mockClearTopicMessages}
+        underReplicatedPartitions={underReplicatedPartitions}
+        inSyncReplicas={inSyncReplicas}
+        replicas={replicas}
+      />
+    );
+
   describe('when it has internal flag', () => {
     it('does not render the Action button a Topic', () => {
       const component = shallow(
```

Comment:
 Do we really need shallow from enzyme here? Or we can rewrite this test with renderComponent from testing library

Reply:
Yeah It looks like easy win

Reply:
Decided to rewrite it in another issue